openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -32,20 +36,24 @@ def course_home_url(course):
     )
 
 
-class TestCourseHomePage(SharedModuleStoreTestCase):
+class CourseHomePageTestCase(SharedModuleStoreTestCase):
```

Comment:
This is not a comment on this PR, but when reviewing names elsewhere, I noticed that none of the other new features have a tests directory.  Does something seem odd about that?